src/main/java/kotlinx/reflect/lite/builtins/JvmAbi.kt

```diff
@@ -0,0 +1,12 @@
+// Partially copied: https://github.com/JetBrains/kotlin/blob/69a13a32691ea3b952f763aa958e276128474718/core/compiler.common.jvm/src/org/jetbrains/kotlin/load/java/JvmAbi.kt
+package kotlinx.reflect.lite.builtins
+
+object JvmAbi {
+    fun getterName(name: String): String = "get" + name.capitalize()
```

Comment:
I see that the original logic is more elaborate, also having special behavior for the `is` prefix. Why is this code different?

Reply:
I've removed it whatsoever, it's unused for now